src/components/Ui/CollapsibleContent.vue

```diff
@@ -0,0 +1,72 @@
+<script>
+import chevronIcon from '@/assets/icons/chevron.svg';
+
+export default {
+  props: ['open', 'title', 'number', 'hideRemove', 'showArrow', 'borderless'],
+  emits: ['toggle'],
+  data() {
+    return { chevronIcon };
+  }
+};
+</script>
```

Comment:
I'd like to encourage you to use composition API. Would be much shorter.

```suggestion
<script setup>
import chevronIcon from '@/assets/icons/chevron.svg';

defineProps(['open', 'title', 'number', 'hideRemove', 'showArrow', 'borderless']);
defineEmits(['toggle']);
</script>
```